cmake/Modules/Findswig.cmake

```diff
@@ -9,15 +9,12 @@ find_package_handle_standard_args(SWIG DEFAULT_MSG
 if(NOT SWIG_EXECUTABLE)
   find_package(Git REQUIRED)
 
-  set(URL https://github.com/swig/swig)
-  set(VERSION fbeb566014a1d320df972aef965daf042db7db36) # Tag origin/rel-3.0.12
-  set_target_description(swig "Simplified Wrapper and Interface Generator (SWIG)" ${URL} ${VERSION})
+  set(URL http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz)
```

Comment:
https seems also working

Reply:
Hmm, link from downloads section of Swig website doesn't contain https so I would prefer to leave it as is.

Reply:
Anyway, it's serious lack for potential MITM attacks on Iroha users, so if HTTPS working - it could be better to enable it.